docs/src/main/paradox/docs/delta/api/files-api.md

```diff
@@ -32,6 +32,18 @@ When using the endpoints described on this page, the responses will contain glob
 - `_incoming`: address to query to obtain the @ref:[list of incoming links](resources-api.md#list-incoming-links)
 - `_outgoing`: address to query to obtain the @ref:[list of outgoing links](resources-api.md#list-outgoing-links)
 
+<a id="custom-file-metadata">
```

Comment:
Why do you need this ?

Reply:
The list of metadata fields were duplicated in several places